### DISCUSSION TITLE
registration is disabled on vaultwarden.discourse.group

### DISCUSSION THREAD

**Discussion by AlexShku** [dani-garcia/vaultwarden#4012] [Q&A] [answered]
Hello! 

I want to ask for help or a hint on the forum but apparently registration is disabled.
I need to change the logic of the organization policy so that Email 2fa is mandatory, or disable everything except Email 2fa.

![image](https://github.com/dani-garcia/vaultwarden/assets/91185761/0d7a43d3-9e9e-417a-825a-baf66c569c9f)

**Comment by BlackDex** [comment]
I just tested it, and works fine. Maybe do what the error message suggest and request a forgot password mail.

Further, you can't make a specific 2fa option mandatory. You can only make 2fa mandatory for all users of a specific organization.

You also can not disable other 2fa options. If you want something like that, you need to hide those options via css. But that doesn't mean smart people can still find those.

**Reply by AlexShku** [reply]
I also thought about css styles but it is not an elegant solution but thanks for the response.

**Reply by AlexShku** [reply]
Could you please advice, where I can find appropriate css file? 
I see only three css files: 
 - bootstrap
 - admin
 - 404
but the are not connected, 

How to reach this file **app/main.f9a460fbbcf515d25766.css** before deploying?

**Reply by BlackDex** [reply]
Checkout this comment: https://github.com/dani-garcia/vaultwarden/discussions/2220#discussioncomment-1938492

**Reply by BlackDex** [reply]
And this one too: https://github.com/dani-garcia/vaultwarden/discussions/1954#discussioncomment-1262398

**Reply by AlexShku** [reply]
Hello @BlackDex! 

When I run the script to copy to a file  `cat /custom.css >> /web-vault/app/main.*.css`, I get a new file - **'main.*css'**.. This is probably due to the fact that the required file does not yet exist at the time the script is launched. I tried to pause the execution, but without results.

I copy the data at the end of the docker file before starting the start.

```
`WORKDIR /

COPY docker/healthcheck.sh /healthcheck.sh
COPY docker/start.sh /start.sh

COPY custom.css /custom.css

COPY --from=vault /web-vault ./web-vault
COPY --from=build /app/target/final/vaultwarden .

HEALTHCHECK --interval=60s --timeout=10s CMD ["/healthcheck.sh"]

COPY init.sh /etc/vaultwarden.sh

CMD ["/start.sh"]`
```

**Reply by BlackDex** [reply, marked as answer]
If you want to do it via a container build, maybe it's best to use tee here instead of only cat.
So, `cat custom.css | tee -a web-vault/app/main.*.css`
Maybe that works.

**Reply by AlexShku** [reply]
Brilliant! That works perfect! Thank you very much!
